tests/resources/BrokenClass.php

```diff
@@ -0,0 +1,11 @@
+<?php
```

Comment:
What about putting this file into `Fixtures/code` instead of in the current location? I would like to drop the `resources` folder someday since it's just a sort of duplicate of the first

Reply:
I did it to leverage the PHPStan exclusion rule; moved and ignored it anyway.